src/axolotl/utils/train.py

```
@@ -0,0 +1,39 @@
+    possible_checkpoints = [str(cp) for cp in Path(cfg.output_dir).glob("checkpoint-*")]
+    if len(possible_checkpoints) > 0:
+        sorted_paths = sorted(
+            possible_checkpoints,
+            key=lambda path: int(path.split("-")[-1]),
+        )
+        if not update:
+            return sorted_paths[-1]
+        last_checkpoint = sorted_paths[-1]
+
+    if cfg.resume_from_checkpoint is None and cfg.auto_resume_from_checkpoints:
+        if last_checkpoint is not None:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)